cd/scripts/release-controller.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+USAGE="
+Usage:
+  $(basename "$0")
+
+Publishes the controller image and helm chart for an ACK service controller as part of prowjob
+when a service controller repository is tagged with a semver format '^v\d+\.\d+\.\d+$'
+See: https://github.com/aws-controllers-k8s/test-infra/prow/jobs/jobs.yaml for prowjob configuration.
+
+Environment variables:
+  REPO_NAME:                Name of the service controller repository. Ex: apigatewayv2-controller
+                            This variable is injected into the pod by Prow.
+  PULL_BASE_REF:            The value of tag on service controller repository that triggered the
+                            postsubmit prowjob. The value will be in the format '^v\d+\.\d+\.\d+$'.
+                            This variable is injected into the pod by Prow.
+  DOCKER_REPOSITORY:        Name for the Docker repository to push to
+                            Default: $DEFAULT_DOCKER_REPOSITORY
+  AWS_SERVICE_DOCKER_IMG:   Controller container image tag
+                            Default: aws-controllers-k8s:<AWS_SERVICE>-<VERSION>
+                            VERSION is calculated from $PULL_BASE_REF
+  QUIET:                    Build controller container image quietly (<true|false>)
+                            Default: false
+  HELM_REGISTRY:            Name for the helm registry to push to
+                            Default: public.ecr.aws/aws-controllers-k8s
+  HELM_REPO:                Name for the helm repository to push to
+                            Default: chart
+"
+
+# find out the service name and semver tag from the prow environment variables.
+AWS_SERVICE=$(echo "$REPO_NAME" | rev | cut -d"-" -f2- | rev | tr '[:upper:]' '[:lower:]')
+VERSION=$PULL_BASE_REF
+
+# Important Directory references based on prowjob configuration.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCRIPTS_DIR=$DIR
+CD_DIR=$DIR/..
+TEST_INFRA_DIR=$CD_DIR/..
+WORKSPACE_DIR=$TEST_INFRA_DIR/..
+SERVICE_CONTROLLER_DIR="$WORKSPACE_DIR/$AWS_SERVICE-controller"
+
+# Check all the dependencies are present in container.
+source "$SCRIPTS_DIR"/lib/common.sh
+check_is_installed buildah
+check_is_installed aws
+check_is_installed helm
+check_is_installed git
+
+if [[ $PULL_BASE_REF = stable ]]; then
```

Comment:
The Git branch will be `$MAJOR_VERSION-stable` I believe.

Reply:
This will be the name of branch, which is 'stable' as per the doc content in community repo. 
> "In the stable Git branch of the ElastiCache service controller's source repository,"

From this variable I create VERSION variable which is $MAJOR_VERSION-stable